data/weapons/weapon_data.csv

```diff
@@ -205,7 +205,7 @@ Thunderbolt SRM Torso,diableavionics_avalancheTorso,0,,400,600,,550,,5,6,5,2,,,H
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 #WARLUST,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 IBIS (mono-barrel),diableavionics_warlusthead,,,,300,,40,,,90,4,,,,FRAGMENTATION,,,0,0.1,1,,0,12,1,20,,1000,,,10,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
-Magicbox Missile Rack,diableavionics_warlustmissile,1,0.75,450,750,,150,,3,15,5,5,0.15,1,FRAGMENTATION,,,0,1.5,,,7,14,14,12,,475,35,3.7,40,,,"PD,PD_ONLY,DO_NOT_AIM,SYSTEM",,,,,,,,,,,,,,,
+Magicbox Missile Rack,diableavionics_warlustmissile,1,0.75,450,750,,150,,3,15,5,8,0.5,1,FRAGMENTATION,,,0,1.5,,,7,14,14,12,,475,35,3.7,40,,,"PD,PD_ONLY,DO_NOT_AIM,SYSTEM",,,,,,,,,,,,,,,
```

Comment:
Warlust Magicbox Missile Rack max ammo set to 8 (from 5). Ammo reload speed set to 0.5 per second (from 0.15 per second).